jsr.json

```diff
@@ -0,0 +1,27 @@
+
+
+{
```

Comment:
Is this an intentional file, or a build output or something?

Reply:
It is a file generated by Speakeasy, if you prefer, we can add it to .gitignore

Reply:
Sounds good, we can ignore!